### DISCUSSION TITLE
How to use the CPU MaxLoad value in the Nuitrack.config file?

### DISCUSSION THREAD

**Discussion by emanuelgollob** [3DiVi/nuitrack-sdk#50] [Q&A]
Hello everyone,
I use the Nuitrack C# sample with the AI skeleton tracker and would like to limit the maximum CPU usage.
I found this setting in the nuitrack.config but couldn´t find any explanation for it:
  
    "CPU": {
        "MaxLoad": -1.0
    },

I guess -1.0 means no restriction at all, but how could I set it to, e.g. 60% of my maximum CPU power?

Thank you!

**Comment by ir4kli09** [comment]
Closing this discussion, answered [here](https://community.nuitrack.com/t/how-to-use-the-cpu-maxload-value-in-the-nuitrack-config-file/3110)
